CompilerSource/settings.h

```diff
@@ -94,7 +94,7 @@ struct CompilerInfo {
 };
 
 extern CompilerInfo compilerInfo;
-
+extern std::string enigma_root;
```

Comment:
How confident are you that this will always be loaded before it's needed? What's our insurance policy?

Actually, seems this is part of our growing wad of static state. Can this be part of CompilerInfo or any of the other lumps of global state I've been trying to consolidate?

Reply:
well compiler info is stuff from your compiler.ey so I dont think it really fits there

Reply:
parse_ide_settings or w/e should always be run before compile and that and compile are only things that need this var afaik